modules/sdk/node-build/esbuild/bundleJavaScriptAndCSS.mjs

```diff
@@ -0,0 +1,62 @@
+import esbuild from 'esbuild';
+import fs from 'fs/promises';
+import path from 'path';
+
+import getExactAliasPlugin from './getExactAliasPlugin.mjs';
+import getFlatName from '../util/getFlatName.mjs';
+import getEntryPoint from './getEntryPoint.mjs';
+import getExternals from './getExternals.mjs';
+import writeExportBridge from './writeExportBridge.mjs';
+import writeImportBridges from './writeImportBridges.mjs';
+
+export default async function bundleJavaScriptAndCSS(globalImports, globalSymbols, projectExports) {
+	await writeImportBridges(globalImports, globalSymbols);
+
+	await Promise.all(
+		projectExports
+			.filter(moduleName => !moduleName.endsWith('.css'))
+			.map(moduleName => bundle(globalImports, projectExports, moduleName))
+	);
+}
+
+async function bundle(globalImports, projectExports, moduleName) {
+	const config = {
```

Comment:
nit: This is a specific esbuild config right? if so, lets name it that(`esbuildConfig`) to not confuse it with some other tool or custom config

Reply:
done